### ISSUE TITLE
New maintainer needed

### ISSUE THREAD

**Issue by jaraco** [jaraco/path.py#77]
The time has come for me to step back and limit my involvement in this project. In order to avoid it being abandoned, I'd like to identify a successor to take over maintenance. I'm happy to mentor a new maintainer and contribute incidentally, but a volunteer is needed to maintain the project. I would like to transfer the repository and assign ownership to the path.py project in PyPI. Please express your interest here (preferred) or message me privately if appropriate.

**Comment by jfmatth** [comment]
I hope someone steps up.

**Comment by sametmax** [comment]
I sent some probes on twitter and reddit (http://www.reddit.com/r/Python/comments/2ttsv1/the_author_of_pathpy_is_looking_for_a_new/).

**Comment by jfmatth** [comment]
I've looked around, and I don't know how it differs from pathlib, but if the two perform basically the same things, you may want to  point people in that direction.  

Pathlib is now in stdlib 3.4+ and is backwards compatible for below.

J

**Comment by aclark4life** [comment]
I can help with releases, etc if you don't find a "real" maintainer. If you want to pursue this, transfer the repo to an org e.g. python-path and invite me.

**Comment by Nodd** [comment]
@jfmatth The main difference is that path inherits from str, so it can be directly passed as argument to any function or library. Also, path.py has more tools compared to pathlib.

**Comment by ubaniak** [comment]
If by Friday no one steps up, I would be happy to maintain this project.

**Comment by msabramo** [comment]
I'm willing to help. Maybe @ubaniak and I could share?

**Comment by ubaniak** [comment]
So Tomorrow I will create a new repo for Path.py. Ill try to get in contact with @jaraco to discuss transitioning the project.

**Comment by jaraco** [comment]
Probably better to move the repo to a new owner or simply add access to the existing repo in order to retain the existing tickets and discussions.

I'm a little wary of creating an org as Alex suggested, as that would seem to suggest that even for a one-module project like path.py, we could end up creating a separate organization.

On the other hand, it is confusing that the project must be owned by somebody.

I just wish there were an umbrella organization for projects like this.

For now, I'll add Alex, Mark, and Ubaniak as authors to the project and maintainers on PyPI.

**Comment by jaraco** [comment]
Thanks to everyone for stepping up.

I've added all three volunteers as collaborators on this project. I've also added Marc and Alex as maintainers on the PyPI service so they can make releases. @ubaniak what is your username on PyPI?

I should now note that the current release process as lightly documented in release.py uses 'jaraco.packaging', which assumes Mercurial is the SCM tool. I imagine most everybody will be using a native git client instead. One of the first tasks in taking ownership of this project will be transitioning that release process to something that supports a native git client.

Right now the release process is automated, handling the tagging, test confirmation, changelog confirmation, building, and releasing of the project. I'd recommend keeping some process to support that. I'd even support extending jaraco.packaging to support git, though I wouldn't recommend it.

In any case, I'll happily work with anyone to help with the transition, so don't hesitate to ask.
